certificates/views.py

```diff
@@ -108,6 +118,15 @@ def get_context_data(self, **kwargs):
         context['name'] = certificate.user.profile.full_name
         context['signatories'] = list(signatories)
         context['certificate'] = certificate
+        context[
+            'linkedin_share_context'
+        ] = 'certId={certId}&issueMonth={issueMonth}&issueYear={issueYear}&name={certName}&organizationId={organizationId}&isFromA2p=true'.format(
+            certId=certificate.hash,
+            issueMonth=certificate.created_on.month,
+            issueYear=certificate.created_on.year,
+            certName=program.title,
+            organizationId="2746406",
```

Comment:
I think it would be a good idea that we use this value through some settings/env variable, Obviously that solely depends upon the probability of this value's change. 
It will give us the freedom of changing this value in the future without any code change & deployment.